python/tank/util/user_settings.py

```diff
@@ -327,7 +327,7 @@ def _to_display_value(self, value):
         """
         if value is None:
             return "<missing>"
-        elif value is "":
+        elif value == "":
```

Comment:
Fixes a SyntaxWarning. `is` should only be used with singleton's like `None` or classes